### PR TITLE
Comment in evaluation

### PR DESCRIPTION
**Pull request by sofipacifico**
**Done
*General  user (Responsible, Calificators and Autoevaluations) can see a comment field where they can add a general comment. In Company case the template is diffetent, the company user can see a comment field with three different types of comments.
*Save data!

### CODE REVIEW

File: source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml [hunk 1]
```diff
@@ -42,10 +42,24 @@
             </div>
             <p class="explanatory-text">*Si no tenes conocimientos para calificar un campo puedes dejarlo vacio</p>
         </div>
+        <!--ko if: userView == 3-->
```
Comment (sofipacifico) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml]: Improve the logic to considerate different evaluation states!

File: source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml [hunk 2]
```diff
@@ -42,10 +42,24 @@
...
+            <div class="comments-container">
+                <div class="title">Fortalezas </div><div class="content-editable" contenteditable="true" data-bind="event: {keyup: onKeyUpComment.bind($data, evaluation.strengthsComment)}"></div>
+                <div class="title">Aspectos a mejorar </div><div class="content-editable" contenteditable="true" data-bind="event: {keyup: onKeyUpComment.bind($data, evaluation.improveComment)}"></div>
+                <div class="title">Plan de acción </div><div class="content-editable" contenteditable="true" data-bind="event: {keyup: onKeyUpComment.bind($data, evaluation.actionPlanComment)}"></div>
+            </div>
+        </div>
+        <h4>Comentarios de la empresa</h4>
+        <div class="comments-container comments-company" contenteditable="true" data-bind="event: {keyup: onKeyUpComment.bind($data, generalComment)}"></div>
+        <!-- /ko -->
+        <!--ko if: userView != 3 -->
```
Comment (sofipacifico) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml]: Improve the logic to considerate different evaluation states!

File: source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml (hunk 1 above)
Comment (joseguti84) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml]: :+1:

File: source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml (hunk 2 above)
Comment (joseguti84) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml]: :+1:

**Pull request merged**